cognite/powerops/custom_modules/power_model_v1/data_models/containers/MultiScenarioMatrix.container.yaml

```diff
@@ -1,26 +0,0 @@
-space: '{{powerops_models}}'
-externalId: MultiScenarioMatrix
-name: MultiScenarioMatrix
-usedFor: node
-properties:
-  production:
-    type:
-      list: true
-      type: timeseries
-    nullable: true
-    autoIncrement: false
-    name: production
-  price:
-    type:
-      list: true
-      type: timeseries
-    nullable: true
-    autoIncrement: false
-    name: price
-constraints:
-  requiredBidMatrix:
-    require:
-      space: '{{powerops_models}}'
-      externalId: BidMatrix
-      type: container
-    constraintType: requires

```

Comment:
Why can't I find `MultiScenarioMatrixRaw.container.yaml` in containers? The type of the `matrix` property needs to be changed from `str` to `Sequence`, at least in the generated Python code.

Reply:
We don't need a separate container for that as it is mainly inheriting from bidmatrix. But I checked that the type for matrix is a sequence there so not sure where the str type comes from.. Where do you see that?

Reply:
I see that [here](https://github.com/cognitedata/power-ops-sdk/blob/1540d9bdbec67cffd63f27c6f9829fdd86adcd18/cognite/powerops/custom_modules/power_model_v1/data_models/containers/BidMatrix.container.yaml#L17) we have got a `sequence` type, but [here](https://github.com/cognitedata/power-ops-sdk/blob/1540d9bdbec67cffd63f27c6f9829fdd86adcd18/cognite/powerops/client/_generated/v1/data_classes/_bid_matrix.py#L119) we've got a `str or None` type.